python-package/xgboost/spark/data.py

```diff
@@ -331,3 +334,25 @@ def split_params() -> Tuple[Dict[str, Any], Dict[str, Union[int, float, bool]]]:
         assert dvalid.num_col() == dtrain.num_col()
 
     return dtrain, dvalid
+
+
+def pred_contribs(
```

Comment:
@trivialfis Do you think it's better to move pred_contribs function to XGBModel?

Reply:
Maybe we can merge it into the `predict` method for consistency?

Reply:
wow, good suggestion.

Reply:
@trivialfis can you file a followup PR to merge it into XGBModel/XGBClassifier.predict method

Reply:
sure.